### ISSUE TITLE
What is the interpretation of nuisance_scores_y and nuisance_scores_t for any DML / DR  based estimation?

### ISSUE THREAD

**Issue by TARSHITA** [microsoft/EconML#601]
This may be a trivial clarification, but if we pass model_y and model_t as stage 1 models  to LinearDML (say model_y=RandomForestRegressor(random_state=123) and model_t=RandomForestClassifier(random_state=123), without tuning for hyperparameters (of stage 1 and stage 2 models), I want to understand : is there any relationship of the nuisance score lists (est.nuisance_scores_y and est.nuisance_scores_t) with the performance of model_y and model_t. 

If yes, what performance metric is default for est.nuisance_scores_y and est.nuisance_scores_t? And can we use them for hyperparameter tuning?

Here's the estimation code for reference:

```
est = LinearDML(model_y=RandomForestRegressor(random_state=123), 
                model_t=RandomForestClassifier(random_state=123),
                discrete_treatment=True, categories=[0, 1],
                fit_cate_intercept=True,
                cv=5,
                random_state=123)
est.fit(Y, T, X=None, W=W,inference='auto') 

```
Will appreciate any help.Thanks in advance!

**Comment by kbattocchi** [comment]
Yes, each model_y and model_t is cloned, and separate clones are then trained on several different folds of the data (in your case, because you specified `cv=5`, there will be 5 folds and each model will be trained on 4/5 of the data).  The nuisance score is computed on the out-of-fold data, using whatever `score` method the estimator provides (mean accuracy for [RandomForestClassifier](https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.RandomForestClassifier.html#sklearn.ensemble.RandomForestClassifier.score) and R^2 for [RandomForestRegressor](https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.RandomForestRegressor.html#sklearn.ensemble.RandomForestRegressor.score)).

It is possible to use the nuisance scores for tuning (e.g. if you train multiple LinearDML estimators using different hyperparameters, then you could look at the nuisance scores and pick the estimator with the best scores on average), but it would be more efficient to do hyperparameter tuning first on the full dataset using normal cross-validation (regressing Y on W for the Y model and T on W for the T model) and then just use those resulting models as inputs to DML, since DML is doing a bunch of additional work.

**Comment by TARSHITA** [comment]
Thanks a lot for clarifying!

**Comment by LuqianSun** [comment]
@kbattocchi hi, is there any reference paper for "use the nuisance scores for tuning"?
